lib/model/message.dart

```diff
@@ -35,6 +35,37 @@ mixin MessageStore {
   /// All [Message] objects in the resulting list will be present in
   /// [this.messages].
   void reconcileMessages(List<Message> messages);
+
+  /// Whether the current edit request, if any, has failed.
+  ///
+  /// Will be null if there is no current edit request
+  /// or the message was deleted.
+  /// Will be false if the current request hasn't failed
+  /// and the update-message event hasn't arrived.
+  bool? getEditMessageErrorStatus(int messageId);
+
+  /// Edits a message's content.
```

Comment:
Given the context of being on [MessageStore] (or [PerAccountStore]), this reads as if it's making an edit to the data we have locally. The fact that it's making an API request over the network is an important part of its meaning, so should be made explicit.

E.g.:
```suggestion
  /// Edit a message's content, via a request to the server.
```